modules/rest-api/pages/rest-manage-log-collection.adoc

```diff
@@ -0,0 +1,50 @@
+= Managing Log Collection
+:description: pass:q[Couchbase-Server logs can be collected, prior to inspection.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+== HTTP method and URI
+
+----
+POST /controller/startLogsCollection
+
+GET /pools/default/tasks
+
+POST /controller/cancelLogsCollection
+----
+
+== Description
+
+The Couchbase Logging facility records important events, and saves the details to log files, on disk.
+By default, on Linux systems, log files are saved to `/opt/couchbase/var/lib/couchbase/logs`; on MacOS, to `/Users/username/Library/Application Support/Couchbase/var/lib/couchbase/logs`; and on Windows, to `C:\Program Files\Couchbase\Server\var\lib\couchbase\logs`.
+On each node within a Couchbase Server-cluster, logging is performed continuously.
+Logged information can be collected either from all or from selected nodes within the cluster.
+
+The `POST /controller/startLogsCollection` method and endpoint start the collection of Couchbase-Server log files.
+The `POST /controller/cancelLogsCollection` method and endpoint can be used to cancel collection, once it has been started.
+The `GET /pools/default/tasks` method and endpoint, which are fully described in xref:rest-api:rest-get-cluster-tasks.adoc[Getting Cluster Tasks], can be used to return status on ongoing tasks, including the collection of log files.
+
+Both `POST /controller/startLogsCollection` and `POST /controller/cancelLogsCollection` require either the *Full Admin* or the *Cluster Admin* roles.
+
+== Curl Syntax
+
+[source,bourne]
+----
+curl -X POST http://<ip-address-or-domain-name>:8091/controller/startLogsCollection
+-u Administrator:password
+-d nodes=[<node-list]
+-d logRedactionLevel=['partial' | 'none']
+-d logDir=<pathname>
+-d tmpDir=<pathname>
+-d logRedactionSalt=<string>
```

Comment:
IMO this shouldn't be documented.  Otherwise more explanation might be needed about what is a salt, how it is used, etc.  Better to just have the value automatically generated.

Reply:
Happy to remove -- however, the salt *is* documented by the CLI; on [this](https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli-collect-logs-start.html) page. So if I remove, we become inconsistent. Or perhaps we remove and file a ticket against the CLI doc.

Reply:
As it's already documented elsewhere I take back my suggestion for removal.